### ISSUE TITLE
CTRL-^ goes back to log buffer instead of previous open buffer

### ISSUE THREAD

**Issue by oryband** [Olical/conjure#163]
`CTRL-^` (or `CTRL-6`) is used to go back to the previous opened buffer. There's an annoying edge case where if you have two buffers you're currently working on (two clojure files), then open Conjure log buffer using `<localleader>ls`, close it using `lq`, and finall press `CTRL-^` - it goes back to the log buffer instead of the previous clojure buffer.

**Comment by Olical** [comment]
Since Conjure's log _was_ the previous opened buffer, I'm not sure how I can get around this? Is there any way to alter the buffer history in nvim? :thinking: I'll have to investigate if this is even possible, if you know of a way to do that, I'll have a read up on it.

I feel like this'd actually be detrimental for a few use cases since I use `<prefix>gd` to jump to the definition of some things from the log then `CTRL-O` to jump back, I wonder if it would break that case.

**Comment by oryband** [comment]
Thing is, current behavior contradicts similar behavior from other plugins - for example if you open a buffer explorer or an fzf buffer and then CTRL-^: it doesn't throw you to that temporary buffer. perhaps Conjure's log buffer could beahve the same.

**Comment by Olical** [comment]
I had to go read up on `keepalt` and `:h alternate-file`, really interesting! I haven't really used it before, but I hooked `keepalt` into the log split / tab creation so it should behave as expected on the `develop` branch now. Let me know if that does the trick for you!

I misunderstood you at first, I was thinking about `CTRL-I` and `CTRL-O` to jump back and forth in the jumplist, which I think is something I can't modify? And I like having the log in there anyway, alternate files are a different thing and I agree with you now :smile: 

Thanks for prompting the improvement!
